packages/core/src/app/payment/PaymentForm.tsx

```diff
@@ -217,6 +217,8 @@ const PaymentMethodListFieldset: FunctionComponent<PaymentMethodListFieldsetProp
                 paymentProviderRadio: getUniquePaymentMethodId(method.id, method.gateway),
                 shouldCreateAccount: true,
                 shouldSaveInstrument: false,
+                accountNumber: '',
+                routingNumber: '',
```

Comment:
Could you please link me the related type/interface changes?

Reply:
https://github.com/bigcommerce/checkout-js/blob/a55aa556b13c75c527e8394374f24947d32e0f3c/packages/payment-integration-api/src/PaymentFormValues.ts#L2